forward_engineering/helpers/viewHelper.js

```diff
@@ -13,36 +13,48 @@ const { getOptions, getPrimaryKeyList } = require('./tableHelper');
 
 const setDependencies = ({ lodash }) => (_ = lodash);
 
-const getColumnNames = (collectionRefsDefinitionsMap, columns) => {
+const getColumnNames = ({ columnsDefinitions, isParentActivated = true }) => {
+	const firstActiveIndex = columnsDefinitions.findIndex(item => item.isActivated);
+
+	return columnsDefinitions.map(({ name, isActivated }, index) => {
+		const statement = `${index === 0 || firstActiveIndex === index ? '' : ','} ${name}`;
+		return commentDeactivatedStatement(statement, isActivated, isParentActivated, INLINE);
+	});
+};
+
+const getColumnDefinitions = (collectionRefsDefinitionsMap, columns) => {
 	return _.uniq(
-		Object.keys(columns).map(name => {
+		Object.entries(columns).map(([name, definition]) => {
```

Comment:
`definition` is misleading here, definition is in the `collectionRefsDefinitionsMap` and here we have a reference, I would call it smth like `referenceData` or `viewColumn`.